assets/src/dashboard/app/views/templateDetails/header/index.js

```diff
@@ -33,6 +33,7 @@ function Header({ onBookmarkClick, onHandleCtaClick }) {
     <Layout.Fixed>
       <DetailViewNavBar
         ctaText={__('Use template', 'web-stories')}
+        closeViewAriaLabel={__('Go to explore templates', 'web-stories')}
```

Comment:
Should this be `Go to Explore Templates`? Or even `Go to %s` as this string might come up more times? Using a placeholder also allows us to more easily change the label without breaking translations

Reply:
Sure, i'll update this to use the page title as seen in the left nav.